Traceback (most recent call last):
  File "/kunden/homepages/8/d657042007/htdocs/CodeSet/private_server/guard/CELL-33/Q-7/33-7.py", line 5, in <module>
    print(count_to)
NameError: name 'count_to' is not defined
